01-guessing_game/NumberGuessing.java

```diff
@@ -0,0 +1,47 @@
+//RandomNumberGuessingGame
+
+
+import java.util.Scanner;
+
+class NumberGuessing {
+  public static void main(String[] args) {
+
+    Scanner sc = new Scanner(System.in);
+
+    int number = 1 + (int)(10 * Math.random());
+
+
+   System.out.println("Welcome to the Random Number Guessing game");
+
+    int K = 5;
+    int i, guess;
+
+    System.out.println("A number is chosen between 1 to 10." +  " Guess the number within 5 tries. ");
+
+
+    for (i = 0; i < K; i++) {
+
+      System.out.println( "Guess the number:");
+      guess = sc.nextInt();
+
+        // If the number is guessed
+        if (number == guess) {
+          System.out.println( "Congratulations!"+ " You guessed the number.");
+          break;
```

Comment:
good for now, but when we go over while loops you wont need to use `break`
using `break` is sort of bad practice...but I like that you found that as a solution for now.

Reply:
alr